src/together/resources/finetune.py

```diff
@@ -207,6 +219,7 @@ def create(
                 For datasets with the "messages" field (conversational format) or "prompt" and "completion" fields
                 (Instruction format), inputs will be masked.
                 Defaults to "auto".
+            from_checkpoint (str, optional): The checkpoint to be used in the fine-tuning.
```

Comment:
Can you copy over the docstring from the CLI, please? Users can use this class directly from their Python code, so they won't see the right docstring otherwise